racer/INSTRUCTIONS.md

```diff
@@ -1,28 +1,40 @@
 1. `sudo service palomad stop`
 2. `sudo service pigeond stop`
-3. Download paloma v1.13.1 from 04/15/2024 release date
+3. Download the latest paloma v1.13.1 from 04/16/2024 release date. You can check that you have the correct version with `palomad version --long | grep commit`. The commit hash is `1143f40382ff1540dc134ceecf11b50af756428a`
```

Comment:
Could we please put the commit hash number in bold and also add instructions to check their palaomd after build with the command `palomad version --long | tail` to confirm they have the correct hash BEFORE starting their node.